### ISSUE TITLE
Project is dead bu what about authors ?

### ISSUE THREAD

**Issue by stefanKP50** [pagekit/pagekit#974]
No security advices, mo more updates... but authors do not update informations about this project ! Not serious !

**Comment by gitgit2019** [comment]
I agree. Very embarrassing for them. Its not even really linked or promoted on their website anymore. SAD. I just found a use-case where I really needed it but I am not going to use some dead vulnerable software.
